NEMO/admin.py

```diff
@@ -293,6 +297,16 @@ def save_model(self, request, obj, form, change):
 				obj.nonrequired_resource_set.set(form.cleaned_data["nonrequired_resources"])
 
 
+
+class AreaAdminForm(MPTTAdminForm):
+	class Meta:
+		model = Area
+		fields = "__all__"
+	area_calendar_color = forms.CharField(
+		required=False, max_length=9,  initial='#88B7CD'
```

Comment:
add missing `,`
remove extra space before initial
```suggestion
		required=False, max_length=9, initial='#88B7CD',
```